packages/core/src/http-middlewares/after/throw-for-status.js

```diff
@@ -3,11 +3,12 @@
 const errors = require('../../errors');
 
 const throwForStatus = (response) => {
-  if (
-    !response.skipThrowForStatus &&
-    response.status >= 400 &&
-    response.status < 600
-  ) {
+  if (response.skipThrowForStatus) {
+    return response;
+  }
+
+  // eslint-disable-next-line yoda
+  if (400 <= response.status && response.status < 600) {
```

Comment:
no-op change to add some clarity to these conditions (including breaking out `if skipThrowForStatus`)